labs/RLSS19_TextWorld.ipynb

```diff
@@ -0,0 +1,2871 @@
+{
```

Comment:
<p>For some reason, in my setting (notebook run locally) the kernel crashes here with the following log:</p><p><br></p><p><code>I 14:33:32.470 NotebookApp] Kernel interrupted: e9a4973f-40a4-4dc5-be79-80ef2a001792</code></p><p><code>terminate called after throwing an instance of 'std::runtime_error'</code></p><p><code>&nbsp;what():&nbsp;expected ) but found 'ident' here:</code></p><p><code>quantized::fake_quantize_per_tensor_affine_forward(Tensor X, float scale, int zero_point, int num_bits = 8, int quant_delay = 0, int iter = 0) -&gt; Tensor</code></p><p><code>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;~~~ &lt;--- HERE</code></p><p><br></p><p><code>[I 14:35:04.148 NotebookApp] KernelRestarter: restarting kernel (1/5), keep random ports</code></p><p><code>kernel e9a4973f-40a4-4dc5-be79-80ef2a001792 restarted</code></p><p><code>Traceback (most recent call last):</code></p><p><code>&nbsp;File "/home/eleurent/anaconda3/lib/python3.7/runpy.py", line 193, in _run_module_as_main</code></p><p><code>&nbsp;&nbsp;&nbsp;"__main__", mod_spec)</code></p><p><code>&nbsp;File "/home/eleurent/anaconda3/lib/python3.7/runpy.py", line 85, in _run_code</code></p><p><code>&nbsp;&nbsp;&nbsp;exec(code, run_globals)</code></p><p><code>&nbsp;File "/home/eleurent/anaconda3/lib/python3.7/site-packages/ipykernel_launcher.py", line 15, in &lt;module&gt;</code></p><p><code>&nbsp;&nbsp;&nbsp;from ipykernel import kernelapp as app</code></p><p><code>&nbsp;File "/home/eleurent/anaconda3/lib/python3.7/site-packages/ipykernel/__init__.py", line 2, in &lt;module&gt;</code></p><p><code>&nbsp;&nbsp;&nbsp;from .connect import *</code></p><p><code>&nbsp;File "/home/eleurent/anaconda3/lib/python3.7/site-packages/ipykernel/connect.py", line 13, in &lt;module&gt;</code></p><p><code>&nbsp;&nbsp;&nbsp;from IPython.core.profiledir import ProfileDir</code></p><p><code>&nbsp;File "/home/eleurent/anaconda3/lib/python3.7/site-packages/IPython/__init__.py", line 55, in &lt;module&gt;</code></p><p><code>&nbsp;&nbsp;&nbsp;from .terminal.embed import embed</code></p><p><code>&nbsp;File "/home/eleurent/anaconda3/lib/python3.7/site-packages/IPython/terminal/embed.py", line 16, in &lt;module&gt;</code></p><p><code>&nbsp;&nbsp;&nbsp;from IPython.terminal.interactiveshell import TerminalInteractiveShell</code></p><p><code>&nbsp;File "/home/eleurent/anaconda3/lib/python3.7/site-packages/IPython/terminal/interactiveshell.py", line 20, in &lt;module&gt;</code></p><p><code>&nbsp;&nbsp;&nbsp;from prompt_toolkit.formatted_text import PygmentsTokens</code></p><p><code>ModuleNotFoundError: No module named 'prompt_toolkit.formatted_text'</code></p><p><br></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/yfletberliac/rlss2019-hands-on/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
<p>Hm, something from low-level (PyTorch internals or cuda, I guess). I'd check version of PyTorch first</p>